packages/panzer/adapters-ioss/src/Panzer_IOSSConnManager.cpp

```diff
@@ -40,21 +40,970 @@
 // ***********************************************************************
 // @HEADER
 
-#include "PanzerAdaptersIOSS_config.hpp"
+// This is a change for Eric.
 
 #include "Panzer_IOSSConnManager.hpp"
-#include "Panzer_IOSSConnManager_impl.hpp"
 
-#include <stdint.h> // for int64_t
+#include <string>
+#include <vector>
+#include <array>
+#include <algorithm>
+#include <limits>
+
+// MPI includes
+#include <mpi.h>
+
+// Teuchos includes
+#include "Teuchos_Comm.hpp"
+#include "Teuchos_EReductionType.hpp"
+#include "Teuchos_RCP.hpp"
+#include "Teuchos_FancyOStream.hpp"
+#include "Teuchos_Assert.hpp"
+#include "Teuchos_TestForException.hpp"
+
+// Kokkos includes
+#include "Kokkos_DynRankView.hpp"
+
+// Shards includes
+#include "Shards_CellTopology.hpp"
+
+// Phalanx includes
+#include "Phalanx_KokkosDeviceTypes.hpp"
+
+// Panzer includes
+#include "Panzer_GeometricAggFieldPattern.hpp"
+#include "Panzer_IntrepidFieldPattern.hpp"
+//#include "Panzer_STK_PeriodicBC_Matcher.hpp"
+//#include "Panzer_STK_SetupUtilities.hpp"
+#include "Panzer_FieldPattern.hpp"
+#include "Panzer_IOSSDatabaseTypeManager.hpp"
+
+// Zoltan includes
+#include "Zoltan2_findUniqueGids.hpp"
+
+// Ioss includes
+#include "Ioss_CodeTypes.h"
+#include "Ioss_ParallelUtils.h"
+#include "Ioss_DatabaseIO.h"
+#include "Ioss_GroupingEntity.h"
+#include "Ioss_Region.h"
+#include "Ioss_EntityBlock.h"
+#include "Ioss_ElementTopology.h"
+#include "Ioss_NodeBlock.h"
+#include "Ioss_ElementBlock.h"
 
 namespace panzer_ioss {
 
-template class IOSSConnManager<int>;
+using Teuchos::RCP;
+using Teuchos::rcp;
+
+IOSSConnManager::IOSSConnManager(Ioss::DatabaseIO * iossMeshDB)
+   : iossMeshDB_(iossMeshDB), iossRegion_(), iossNodeBlocks_(), iossElementBlocks_(),
```

Comment:
You don't need to invoke default constructors of members (e.g., `iossRegion_()`) explicitly.